applications/openshift/kubernetes/master/api-server/admission_control_plugin-EventRateLimit/rule.yml

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+title: 'Ensure that the admission control plugin EventRateLimit is set'
+
+description: |-
+    'To limit the rate at which the API Server accepts requests, follow
+    the OpenShift documentation and set the desired limits in a configuration
+    file. Then, edit the API Server pod specification file
+    <tt>/etc/kubernetes/manifests/kube-apiserver.yaml</tt> and set the
+    below parameters:
+
+    <pre>--enable-admission-plugins=...,EventRateLimit,...<br />
```

Comment:
Doesn't `<br />` actually show up in the `<pre/>` output? Thought you don't need `<br/>` in `<pre/>`